### DISCUSSION TITLE
Issues with social images on quarto.org 🤔 (maybe not specific though)

### DISCUSSION THREAD

**Discussion by mcanouil** [quarto-dev/quarto-cli#3023] [Q&A] [answered]
I don't know if it's really a bug or some kind of misconfiguration, but for both Twitter and OpenGraph cards, the image is not found or none are defined.

See for example <https://twitter.com/MickaelCanouil/status/1584884666913345536>.
EDIT: the search mechanism to find a preview image (or the propagation of it) seems to take longer than it usually does compared to when the image is set.

Also, there might be something wrong (or not clear at least) in the documentation: <https://quarto.org/docs/websites/website-tools.html#twitter-cards>.

| Option | Description |
|:-------|:------------|
| image | The path to a preview image for this content. By default, Quarto will use the image `value` from the `site:` metadata. If you provide an image, you may also optionally provide an `image-width` and `image-height` to improve the appearance of your Twitter Card. If image is not provided, Quarto will automatically attempt to locate a preview `image`. For more information, see Preview Images. |

Here, the description talks about `site:`, is it really this option or is it `website:`?
It's a bit confusing because there is a `site:` option listed in the same table.

| Option | Description |
|:-------|:------------|
|site | @username of website. Note that strings with special characters such as @ must be quoted in yaml.|

If it's indeed `website:` and not `site:`, then the default image that should be taken for Twitter and OpenGraph cards should be the one in <https://github.com/quarto-dev/quarto-web/blob/4a665d2e5082a7e80a3bb566b3924673f1de7f45/_quarto.yml#L9-L11>, but that's not the case.

**Comment by dragonstyle** [comment, marked as answer]
I think that is just wrong now - we used to look at the site wide image for the social metadata but we no longer do. We stopped doing this as we felt like the experience of using the same image for every page of a website wasn't good.

We really only use that image in feeds right now. I'll open and issue to track correcting the documentation on this, thanks for catching it.

**Reply by mcanouil** [reply]
Thanks for the answer.
I'll mark this as answered and follow on the issue.
